service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ChildControllerMidEventTest.java

```diff
@@ -32,19 +31,21 @@
 
     private static final String AUTH_TOKEN = "Bearer token";
     private static final String USER_ID = "1";
-    private static final ObjectMapper MAPPER = new ObjectMapper();
     private static final String ERROR_MESSAGE = "Date of birth cannot be in the future";
 
     @Autowired
     private MockMvc mockMvc;
 
+    @Autowired
+    private ObjectMapper mapper;
+
     @Test
     void shouldReturnDateOfBirthErrorWhenFutureDateOfBirth() throws Exception {
         CallbackRequest request = CallbackRequest.builder()
             .caseDetails(CaseDetails.builder()
                 .id(12345L)
                 .data(ImmutableMap.of("children1", ImmutableList.of(
-                    createChildrenElement(ZonedDateTime.now().plusDays(1)))))
+                    createChildrenElement(LocalDate.now().plusDays(1)))))
```

Comment:
Are the refactoring changes required to be in scope of this PR?

Reply:
No, annoyingly were already done when I picked up the branch. Not inclined to revert due to extra effort 🤷‍♂

Reply:
Agree, will resolve those.